src/utils/hooks/useScrollPosition.ts

```diff
@@ -0,0 +1,120 @@
+import React from 'react';
```

Comment:
Do we need an accurate scroll position here?
For scrolling to static sections anchors are usually used